chore: getting started doc improvements (#233)
IBM/openapi-validator: README.md

  * [Default mode](#default-mode)
    + [Default values](#default-values)
  * [Spectral configuration](#spectral-configuration)
    + [Changing Spectral rule severity](#changing-spectral-rule-severity)
    + [Custom Spectral rules](#custom-spectral-rules)
- [Warnings Limit](#warnings-limit)
- [Turning off `update-notifier`](#turning-off-update-notifier)
- [License](#license)

<!-- tocstop -->

## Getting Started

> The validator analyzes your API definition and reports any inconsistencies within the API definition. The validator is highly customizable, and supports both OpenAPI 3.0, OpenAPI 2.0 (Swagger 2.0), and Spectral rules.  You can easily create new rules and modify existing ones so that the tool will ensure consistency to your standards.

Reviewer: ```suggestion
The validator analyzes your API definition and reports any inconsistencies within the API definition. The validator is highly customizable, and supports both OpenAPI 3.0, OpenAPI 2.0 (Swagger 2.0), and Spectral rules.  You can easily create new rules and modify existing ones so that the tool will ensure compliance to your standards.
```

The default configuration uses both OpenAPI 3.0 rules as well as Spectral rules. The [default mode](#default-mode) section decscribes these rules. Get started by [installing the tool](#installation), then [run the tool](#usage) on your API definition.  

### Customization

> You can begin to modify the behavior of the validator for your project to meet your prefered standards. Specific validation "rules" can be turned off, or configured to trigger an error, warning, info, or hint message in the validator output.

Reviewer: ```suggestion
You can modify the behavior of the validator for your project to meet your preferred standards. Specific validation "rules" can be turned off, or configured to trigger an error, warning, info, or hint message in the validator output.
```

> Some validations can be configured even further, such as switching the case convention to validate against for parameter names.

Reviewer: ```suggestion
Some validations can be configured even further, such as switching the case convention for parameter names.
```
